libs/MVS/SceneDensify.cpp

```diff
@@ -29,8 +29,7 @@
 *      containing it.
 */
 
-#include "Common.h"
-#include "Scene.h"
+#include "SceneDensify.h"
```

Comment:
this might not work on Windows
"Common.h" is a precompiled header and needs to be always added

Reply:
Okay, I'll check and test compilation on Windows. Thanks.

Reply:
After changing the include structure I was able to build OpenMVS on Windows.